src/main/resources/application.properties

```diff
@@ -24,4 +24,17 @@ orders.cleanup.not-payed.cronexpr=0 0 * * * *
 
 orders.cleanup.not-confirmed.timer-minutes=10
 # 1440 minutes = 24 hours
-orders.cleanup.not-payed.timer-minutes=1440
\ No newline at end of file
+orders.cleanup.not-payed.timer-minutes=1440
+
+
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+
+spring.shopLink.order=http://localhost:8080/order/
+
+# How can I hide this data (mail and/or password) in git/gitHub?
```

Comment:
Komentarz do usunięcia
Możesz zostawić propertas os hasła, ale BEZ hasła, żeby zamarkować, że to hasło jest potrzebne ;)
Albo w haśle możesz nawet jak wół napisać, że tam musi wpaść jakieś realne hasło

Reply:
mam juz w opcjach Intelij przy profilu odpalania